### PR TITLE
Update dashboard tab alert options

### PR DESCRIPTION
**Pull request by maryyliu**
Updated the Dashboard Tab Alert Options in config.proto to include num_passes_to_disable_alert, subject, debug_url, debug_message.
Updated README.md to reflect these changes.

### CODE REVIEW

File: testgrid/README.md [hunk 1]
```diff
@@ -162,7 +162,10 @@ and/or `alert_stale_results_hours` (alerts when tests haven't run recently).
 You can also set `num_passes_to_disable_alert`.
 
 In DashboardTab, set `alert_mail_to_addresses` (comma-separated list of email
-addresses to send mail to).
+addresses to send mail to). Additional options for alerts are `num_passes_to_disable_alert`
```
Comment (michelle192837) [inline testgrid/README.md]: nit: Make this a list of bullet points?

Comment (maryyliu) [reply testgrid/README.md]: updated

**Comment by michelle192837** [comment]
/lgtm
/approve

Awesome :D Thanks!

**Pull request merged**